framework/cmm-interface.md

```diff
@@ -85,17 +85,10 @@ it MUST return [encryption materials](#structures.md#encryption-materials) appro
 
 The encryption materials returned MUST include the following:
 
-- [Algorithm Suite](#algorithm-suite.md)
-    - If the encryption materials request contains an algorithm suite, the encryption materials returned SHOULD contain the same algorithm suite.
-- Plaintext Data Key
-- [Encrypted Data Keys](#structures.md#encrypted-data-keys)
-    - Every encrypted data key in this list MUST correspond to the above plaintext data key. 
+- [Data Key Materials](#structures.md#data-key-materials.md)
+    - If the encryption materials request contains an algorithm suite, the data key materials returned SHOULD contain the same algorithm suite.
```

Comment:
I've heard the CMM interface is permitted to elect to use a different algorithm suite than the one requested. This was implicitly allowed in the stateful version, since there was no requirement that said it couldn't be mutated.